src/CoreBundle/private/modules/CMSTemplateEngine/CMSTemplateEngine.class.php

```diff
@@ -108,18 +111,29 @@ public function Init()
         parent::Init();
 
         $this->sPageId = $this->getPageIdFromRequest();
+
+        // TODO what about language?
```

Comment:
language is not relevant here for loading because we do not use the objects data, default language is o.k.

Reply:
ok